source/pool/test/unit.js

```diff
@@ -1,607 +1,653 @@
+
+    feeToken = await deployMockContract(deployer, IERC20.abi)
+    await feeToken.mock.approve.returns(true)
+    feeToken2 = await deployMockContract(deployer, IERC20.abi)
+
+    stakeContract = await (
+      await ethers.getContractFactory(STAKING.abi, STAKING.bytecode)
+    ).deploy(feeToken.address, 'StakedAST', 'sAST', 100, 10)
+    await stakeContract.deployed()
+
+    pool = await (
+      await ethers.getContractFactory('Pool')
+    ).deploy(CLAIM_SCALE, CLAIM_MAX, stakeContract.address, feeToken.address)
+    await pool.deployed()
+
+    tree = generateTreeFromData({
+      [alice.address]: ALICE_SCORE,
+      [bob.address]: BOB_SCORE,
+      [carol.address]: CAROL_SCORE,
+    })
+  })
+
+  describe('Test constructor', async () => {
+    it('constructor sets values', async () => {
+      const storedScale = await pool.scale()
+      const storedMax = await pool.max()
+      const stakingContract = await pool.stakingContract()
+      const stakingToken = await pool.stakingToken()
+      const adminOwner = await pool.admins(deployer.address)
+      expect(storedScale).to.equal(CLAIM_SCALE)
+      expect(storedMax).to.equal(CLAIM_MAX)
+      expect(stakingContract).to.equal(stakeContract.address)
+      expect(stakingToken).to.equal(feeToken.address)
+      expect(adminOwner).to.equal(true)
+    })
+
+    it('constructor reverts when percentage is too high', async () => {
+      await expect(
+        (
+          await ethers.getContractFactory('Pool')
+        ).deploy(CLAIM_SCALE, 101, stakeContract.address, feeToken.address)
+      ).to.be.revertedWith('MAX_TOO_HIGH')
+    })
+
+    it('constructor reverts when scale is too high', async () => {
+      await expect(
+        (
+          await ethers.getContractFactory('Pool')
+        ).deploy(78, CLAIM_MAX, stakeContract.address, feeToken.address)
+      ).to.be.revertedWith('SCALE_TOO_HIGH')
+    })
+  })
+
+  describe('Test enable', async () => {
+    it('enable successful', async () => {
+      const root = getRoot(tree)
+      await expect(await pool.connect(deployer).enable(root)).to.emit(
+        pool,
+        'Enable'
+      )
+      expect(await pool.roots(root)).to.equal(true)
+    })
+
+    it('enable reverts', async () => {
+      const root = getRoot(tree)
+      await expect(await pool.connect(deployer).enable(root)).to.emit(
+        pool,
+        'Enable'
+      )
+      await expect(pool.connect(deployer).enable(root)).to.be.revertedWith(
+        'ROOT_EXISTS'
+      )
+    })
+  })
+
+  describe('Test staking variables', async () => {
+    it('set stake contract successful', async () => {
+      await pool.connect(deployer).setStakingContract(stakeContract.address)
+      expect(await pool.stakingContract()).to.equal(stakeContract.address)
+    })
+
+    it('set stake contract reverts', async () => {
+      await expect(
+        pool
+          .connect(deployer)
+          .setStakingContract('0x0000000000000000000000000000000000000000')

```

Comment:
I know this address 0x000... was there from a previous commit, though this is a good opportunity to replace it with a variable from @airswap/constants.
 I suggest importing the zero address constant into the test file, and using the constant in place of the current 0x000...
 `const { ADDRESS_ZERO } = require('@airswap/constants')`
 You can see how this is used in other test files like ./source/swap/test/unit.js